.github/workflows/release.yml

```diff
@@ -32,23 +32,31 @@ jobs:
           echo "::set-output name=version::$(git describe --tags --always --dirty | cut -c2-)"
 
   builder:
+    name: builder-${{matrix.os}}-${{matrix.arch}}
     needs: [args]
+    strategy:
+      matrix:
+        os:
+          - linux
+        arch:
+          - amd64
+          - arm64
     permissions:
       actions: read # For the detection of GitHub Actions environment.
       id-token: write # For signing.
       contents: write # For asset uploads.
     uses: slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@v1.4.0
     with:
       go-version: 1.18
-      config-file: .github/config-release.yml
+      config-file: .slsa-goreleaser-${{matrix.os}}-${{matrix.arch}}.yml
       compile-builder: true
       evaluated-envs: "VERSION:${{needs.args.outputs.version}}"
 
   # In case this fails, e.g. build configuration changes, file an issue in slsa-verifier
   if-failed:
```

Comment:
@laurentsimon @asraa Do we even need this in the release workflow? It's not a pre-submit so I don't think we need it to create GitHub issues?

Reply:
I think you're right we can remove the `if-failed`. @suzuki-shunsuke please remove it and let's merge.

Reply:
Removed. bec4a0b8f053af283c72deffffcb5afb3637193f